variables.tf

```
@@ -186,6 +186,24 @@ variable "auto_verified_attributes" {
   ]
 }
 
+variable "account_recovery_setting_recovery_mechanisms" {
+  type        = any
+  description = "(Optional) Account recovery mechanisms"
```

if we change the interface the text could be as follow.

I guess it makes it easier to understand for the users of the module?

```suggestion
  description = "(Optional) A list of Account Recovery Options order by priority. Valid values are verified_email, verified_phone_number, and admin_only."
```